### PR TITLE
Adding web3AuthNetwork in Constructor 

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

File: web-core-sdk/browser-extensions/chrome-extension-web3auth-example/src/App.tsx [hunk 1]
```diff
@@ -29,13 +29,10 @@ function App() {
...
           },
+          web3AuthNetwork: "cyan",
         });
 
-        const openloginAdapter = new OpenloginAdapter({
-          adapterSettings: {
-            network: "cyan",
-          },
-        });
+        const openloginAdapter = new OpenloginAdapter({});
```
Comment (shahbaz17) [inline web-core-sdk/browser-extensions/chrome-extension-web3auth-example/src/App.tsx:35]: Why we're keeping this line @Saviour1001?

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: web-core-sdk/browser-extensions/chrome-extension-web3auth-example/src/App.tsx (hunk 1 above)
Comment (shahbaz17) [reply web-core-sdk/browser-extensions/chrome-extension-web3auth-example/src/App.tsx:35]: It's better to remove it. wdyt?

Comment (Saviour1001) [reply web-core-sdk/browser-extensions/chrome-extension-web3auth-example/src/App.tsx]: removed

**Review by shahbaz17 — APPROVED** [review]
LGTM

**Pull request merged**